packages/trader/src/App/Components/Elements/ContractAudit/contract-details.tsx

```diff
@@ -77,7 +99,7 @@ const ContractDetails = ({ contract_end_time, contract_info, duration, duration_
                             id='dt_commission_label'
                             icon={<Icon icon='IcContractCommission' size={24} />}
                             label={localize('Commission')}
-                            value={<Money amount={commission} currency={currency} show_currency />}
+                            value={<Money amount={commission || 0} currency={currency} show_currency />}
```

Comment:
I think we shouldn't alter the logic for things like commissions here to fulfil typescript rule. Since the value is received from API, we should always show what it is and follow the current logic in case something goes wrong. If we show 0 instead, this will mislead clients and might result in legal consequences.